libs/designsystem/page/src/page.component.scss

```
@@ -248,6 +248,7 @@ ion-content {
     width: 100%;
     position: absolute;
     bottom: 0;
+    transform: none;
```

```suggestion
    transform: initial;
```
Consider setting this to initial to get the original user agent style